tests/FormTest.php

```diff
@@ -101,7 +101,7 @@ public function testFormSubmit()
 
     public function testTextArea()
```

Comment:
we should fix case everywhere (ie. Textarea)

Reply:
@georgehristov wdyt? `TextArea`, `DropDown`, ... Probably everywhere expect `\atk4\ui\DropDown` which should be fixed right after we merge this in another PR